### ISSUE TITLE
Where did the new snake images come from?

### ISSUE THREAD

**Issue by Metallicow** [adafruit/awesome-micropythons#4]
Jon and I did a lot of the work with the original images and conversions around/after the kickstarter for the official micropython repo.
I'm curious where the last 5 snakes in the cartoon style on the readme page came from? and or who drawn them?

**Comment by ptorrone** [comment]
@Metallicow we made these at adafruit when we made the art after getting permission from damien for the badges/stickers (https://www.adafruit.com/product/3270)

here they all are! 
https://www.dropbox.com/sh/f0y50vaphznai8v/AAAla6eZUyAh0fQgTXUlTo-Wa?dl=0

**Comment by Metallicow** [comment]
@ptorrone Cool Thanks. If these are for the public use, could you send a PullREQ to the official repo in `micropython/logo/` directory so others can use them also? There was at one point some community contributions for various logos and whatnot which ended up in the logo/imgs directory.
I noticed the art style was pretty similar to damiens original doodle, so it made me curious since they was not found there lol.

**Comment by Metallicow** [comment]
Ah... they are jpgs. I figured they was .png with alpha. Do you got source files so lossless .pngs might be made?
Edit: nevermind, I see the sourcefile is a .ai Thats a adobe format...

**Comment by ptorrone** [comment]
@Metallicow i can provide any format, here is the .ai, what else ya want?
https://www.dropbox.com/sh/f0y50vaphznai8v/AAAla6eZUyAh0fQgTXUlTo-Wa?dl=0&preview=MicroPython_black_white_backgrounds.ai

**Comment by Metallicow** [comment]
Well, it really isn't a big deal since you have an .ai, but since this is opensource, a .kra(Krita) or GIMP file is what is usable for me nowadays. I made a decent donation to krita since 4.2 (for fixing selections; was an stopper issue for many PS users) so I use that now instead of photoshop as my main Art/Image editor. I don't think that maybe the sourcefile would the best if contributing to the main micropython repo as an .ai would be useful, but a Krita or GIMP file would.

* I just don't have my old Photoshop installed on my machine atm..., that's the small issue, not that I couldn't use it... eventually.

The .dxf and .svg Jon did was what was originally used for the manufacturing process for the original boards and the colored .svgs that other users made added gradienting too.

Basically I am looking for lets say 5 sourcefiles preferably in a open format, and 5 .png (512x512)renders that are optimized that could be added to the official repo(so educators/makers/etc.. could use them easily)

What would the attribution uses for the images be? I know that in images, unless using CC0 basically you use by attribution(which means users have to attribute the original artist). It all comes down to "hey you made a buck off my work, do you credit me?". With the work we did on the original 'we care' but we really don't care in the sense as long as you don't claim the work as your own, We did the artwork, so as long as you dont claim that your ok type of thing. Most of this can be referenced in the commits anyhow.

**Comment by ptorrone** [comment]
@Metallicow all good, here's is a SVG and a PNG of the .ai file:
https://www.dropbox.com/sh/ihjxmyh8y7kkcyd/AAB9lkJ3INzdLUNjKAIiYCNTa?dl=0

as far as attribution, we made these for damien and the micropython project, so whatever damien says is cool with us.

**Comment by Metallicow** [comment]
Ok, but I mean like should I attribute you personally or for example `Adafruit`?

**Comment by ptorrone** [comment]
oh! adafruit!

**Comment by Metallicow** [comment]
kk. Thanks

**Comment by Metallicow** [comment]
Seen the video. Good that you folks are donating a portion of the stickers to Damien. It should be that way. Saw your laptop in the video and recently had a conversation with Pillow about "sticker graffiti" https://github.com/python-pillow/Pillow/issues/4361 . Didn't know stickers was that big a thing on laptops.
The breadboard and things you folks sent way back when still work fine for my board! :) ... tho my micropy board is pinned backwards(Jon said it would be worth more that way lol, for whatever reasons I do not know. He is no longer with us...)
